src/main/java/se/lth/cs/connect/routes/Account.java

```diff
@@ -141,6 +147,27 @@ protected void setup(PippoSettings conf) {
             }
         });
 
+        // GET api.serp.se/v1/account/friends HTTP/1.1
+        GET("/friends", (rc)->{
+        	final JcNode coll = new JcNode("coll");
+        	JcNode u = new JcNode("u");
+        	//find all emails of users in mutual collections
+        	String email = rc.getParameter("email").toString();
+        	List<GrNode> res = Database.query(rc.getLocal("db"), new IClause[]{
+    	            MATCH.node().label("user").property("email").value(email)
+    	                .relation().type("MEMBER_OF").node(coll).label("collection").
+    	                relation().type("MEMBER_OF").node(u).label("user"),
+    	            RETURN.DISTINCT().value(u)
```

Comment:
You could make an additional efficiency boost by returning email addresses instead of user nodes. But I think the current solution is fine so I'll merge it. Good job!